share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -160,6 +160,15 @@ macro(rocm_create_package)
     if(ROCM_VERSION_NUM)
         set(CPACK_PACKAGE_VERSION "${CPACK_PACKAGE_VERSION}.${ROCM_VERSION_NUM}")
     endif()
+    # Set the License to rpm header
+    if(PARSE_LICENSE)
+        set( CPACK_RPM_PACKAGE_LICENSE "${PARSE_LICENSE}" )
+    endif()
+    # Setting the CPACK_RESOURCE_FILE_LICENSE variable.
+    # Installing license file to /opt/rocm/share/doc/<component_name> will happen in rocm_setup_license macro
+    if(PARSE_LICENSE_PATH)
+        set(CPACK_RESOURCE_FILE_LICENSE "${PARSE_LICENSE_PATH}")
+    endif()
```

Comment:
We should add an install statement to install the license file.

Reply:
The installation is done in the `rocm_setup_license` macro added in #63.

Reply:
Users are supposed to call that? There is no documentation for it.

Reply:
It's not intended to be called by users, it's called in `rocm_set_comp_cpackvar`.